server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/eq/utils/HazardUtil.java

```diff
@@ -109,20 +109,12 @@
     // TODO: This json with period is getting hard to read. Move to a static json file - should we use the demand type and unit
     //  definition constants defined here?.
     public final static JSONObject EARTHQUAKE_THRESHOLDS = new JSONObject("{ " +
-        "'" + PGA + "': {'0.0' : null}," +
-        "'" + PGV + "': {'0.0' : null}," +
-        "'" + PGD + "': {'0.0' : null}," +
-        "'" + SA + "':{'0.2': null," +
-        "'0.3': null," +
-        "'1.0': null," +
-        "'0.0': null," +
-        "}," +
-        "'" + SD + "':{'0.2': null," +
-        "'0.3': null," +
-        "'1.0': null," +
-        "'0.0': null," +
-        "}," +
-        "'" + SV + "': {'0.0' : null}," +
+        "'" + PGA + "': {'value': 0.01, 'unit': 'g'}," +
+        "'" + PGV + "': {'value': 0.01, 'unit': 'in/sec'}," +
```

Comment:
I know the wiki says in/sec, but everyone in the code (and fragilities) I believe we are using in/s. We either need to change this to in/s or we'll need to update the convert hazard to support the threshold unit as in/sec when converting since it's matching on the string.

Reply:
This also had to be updated in `commondb.Demand` collection. I'll make sure to make this change on dev and prod when this is being deployed. PGV ntry is using `/sec` instead of `/s`

Tagging @navarroc @longshuicy so you are also aware that this change is needed when deploying